src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,44 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int matrixLength = matrix.length;
```

Comment:
matrixLength not shorter than matrix.length. Do we need this variable?

Reply:
done